src/devices/cpu/z80/z80.h

```diff
@@ -291,6 +292,7 @@ class z80_device : public cpu_device, public z80_daisy_chain_interface
 	const uint8_t *   m_cc_xy;
 	const uint8_t *   m_cc_xycb;
 	const uint8_t *   m_cc_ex;
+	uint8_t           m_mtm_cycles = 3;
```

Comment:
Please don’t initialise data members of devices in the header.  It makes things more fragile, causing unnecessary recompiles, and it’s confusing to read when member initialisations are split between the declarations in the header and the constructor initialiser list in the source file.

Reply:
Ok, makes sense. I have moved it to the constructor.